<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 137, 103);
background-color: rgb(85, 85, 127);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>711</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 127);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>gestion personnel</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>691</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="title">
          <string>ajouter personnel</string>
         </property>
         <widget class="QWidget" name="formLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>39</y>
            <width>321</width>
            <height>201</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>cin</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditcin">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>nom</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditnom">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>prenom</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditprenom">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>mobile</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditmobile">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);
</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>departement</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>date_naissance</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QDateEdit" name="dateEditdate_naissance">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);
</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>salaire</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditsalaire">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboBox">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 85, 127);</string>
             </property>
             <item>
              <property name="text">
               <string>finance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ressources humaines</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>administration</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>info</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>marketing</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>260</y>
            <width>321</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>modifier</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="title">
          <string>afficher personnel</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>341</width>
            <height>241</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableView" name="tableView_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 127);
background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>270</y>
            <width>341</width>
            <height>31</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_6">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>stat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_10">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>creer un pdf</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
      <property name="title">
       <string>TRI</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>221</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableView" name="tableViewT">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 225, 164);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>cin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_salaire">
          <property name="text">
           <string>salaire</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>431</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="title">
          <string>RECHERCHE</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>10</y>
            <width>161</width>
            <height>97</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>nom</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>prenom</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>mobile</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>120</y>
            <width>131</width>
            <height>23</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>valider</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>251</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 225, 164);</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>gestion departement</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>681</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="title">
          <string>ajouter departement</string>
         </property>
         <widget class="QWidget" name="formLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>321</width>
            <height>101</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;Palatino Linotype&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>nom</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>reference</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_reference">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>nb_employes</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_nb_employes">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 85, 127);</string>
             </property>
             <item>
              <property name="text">
               <string>finance</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>marketing</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ressources humaines</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>administration</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>info</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>250</y>
            <width>321</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="ajouter2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>ajouter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_8">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="modifier_departement">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>modifier</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 170, 127);</string>
         </property>
         <property name="title">
          <string>afficher departement</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>19</y>
            <width>321</width>
            <height>221</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTableView" name="tableViewD">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 225, 164);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>321</width>
            <height>61</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="supprimer_departement">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 170, 255);</string>
             </property>
             <property name="text">
              <string>supprimer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
